cli/upgrade.ts

```diff
@@ -65,10 +65,11 @@ export async function upgrade(dev = false, version?: string) {
 
 async function updateDenoConfig(url: URL) {
   try {
-    await Promise.all([
+    await Promise.any([
       Deno.stat("deno.json"),
-      Deno.stat("import_map.json"),
+      Deno.stat("deno.jsonc"),
     ]);
+    await Deno.stat("import_map.json");
```

Comment:
I think this needs to read from Deno config, but let's address it in another PR.